internal/utils/utils.go

```diff
@@ -13,3 +18,24 @@ func HasFinalizer(cr *metav1.ObjectMeta, name string) bool {
 	}
 	return false
 }
+
+// NormalizeLabel normalizes label string according to k8s format
+func NormalizeLabel(name string) (string, error) {
+	// convert name to lowercase and replace '.' with '-'
+	name = strings.ToLower(name)
+	name = strings.Replace(name, ".", "-", -1)
```

Comment:
isn't '.' is valid in label name?

Reply:
There is nothing in the docs about `:` being not allowed.

Reply:
I meant that dot '.' is valid. As well as all the other characters we are trying to replace.

Reply:
ack, fixed